Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,16 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+function CardBanner(props) {
+  return (
+    <React.Fragment>
```

Comment:
hey Novina, `Fragments let you group a list of children without adding extra nodes to the DOM.` why are you using it here? https://reactjs.org/docs/fragments.html